### ISSUE TITLE
F0118 19:56:37.267200 17064 upgrade_proto.cpp:1063] Check failed: ReadProtoFromTextFile(param_file, param) Failed to parse SolverParameter file: C:/Users/Administrator/Desktop/data/train-test/cifar10_quick_solver.prototxt *** Check failure stack trace: ***

### ISSUE THREAD

**Issue by BigVast** [BVLC/caffe#6184]
When I type command "SET GLOG_logtostderr=1
D:/caffe-master/Build/x64/Debug/caffe.exe train --solver=C:/Users/Administrator/Desktop/data/train-test/cifar10_quick_solver.prototxt
pause",I just get the errors below.
I used windows 10 to compile the "caffe",but I got the errors below:
![1](https://user-images.githubusercontent.com/31783797/35097196-1f075154-fc8a-11e7-9a60-3c0c574eb7d6.png)

And my net is just like this:
name: "LeNet"
layer {
  name: "mnist"
  type: "Data"
  top: "data"
  top: "label"
  include {
    phase: TRAIN
  }
  transform_param {
    mean_file:"C:/Users/Administrator/Desktop/data/mean.binaryproto"
    scale: 0.00390625
  }
  data_param {
    source: "C:/Users/Administrator/Desktop/data/mtrainldb"
    batch_size: 25
    backend: LEVELDB
  }
}
layer {
  name: "mnist"
  type: "Data"
  top: "data"
  top: "label"
  include {
    phase: TEST
  }
  transform_param {
    mean_file:"C:/Users/Administrator/Desktop/data/mean.binaryproto"
    scale: 0.00390625
  }
  data_param {
    source: "C:/Users/Administrator/Desktop/data/mtestldb"
    batch_size: 5
    backend: LEVELDB
  }
}
layer {
  name: "conv1"
  type: "Convolution"
  bottom: "data"
  top: "conv1"
  param {
    lr_mult: 1
  }
  param {
    lr_mult: 2
  }
  convolution_param {
    num_output: 6
    kernel_size: 5
    stride: 1
    weight_filler {
      type: "xavier"
    }
    bias_filler {
      type: "constant"
    }
  }
}
layer {
  name: "pool1"
  type: "Pooling"
  bottom: "conv1"
  top: "pool1"
  pooling_param {
    pool: MAX
    kernel_size: 2
    stride: 2
  }
}
layer {
  name: "conv2"
  type: "Convolution"
  bottom: "pool1"
  top: "conv2"
  param {
    lr_mult: 1
  }
  param {
    lr_mult: 2
  }
  convolution_param {
    num_output: 16
    kernel_size: 5
    stride: 1
    weight_filler {
      type: "xavier"
    }
    bias_filler {
      type: "constant"
    }
  }
}
layer {
  name: "pool2"
  type: "Pooling"
  bottom: "conv2"
  top: "pool2"
  pooling_param {
    pool: MAX
    kernel_size: 2
    stride: 2
  }
}
layer {
  name: "ip1"
  type: "InnerProduct"
  bottom: "pool2"
  top: "ip1"
  param {
    lr_mult: 1
  }
  param {
    lr_mult: 2
  }
  inner_product_param {
    num_output: 120
    weight_filler {
      type: "xavier"
    }
    bias_filler {
      type: "constant"
    }
  }
}
layer {
  name: "relu1"
  type: "ReLU"
  bottom: "ip1"
  top: "ip1"
}
layer {
  name: "ip2"
  type: "InnerProduct"
  bottom: "ip1"
  top: "ip2"
  param {
    lr_mult: 1
  }
  param {
    lr_mult: 2
  }
  inner_product_param {
    num_output: 728
    weight_filler {
      type: "xavier"
    }
    bias_filler {
      type: "constant"
    }
  }
}
layer {
  name: "accuracy"
  type: "Accuracy"
  bottom: "ip2"
  bottom: "label"
  top: "accuracy"
  include {
    phase: TEST
  }
}
layer {
  name: "loss"
  type: "SoftmaxWithLoss"
  bottom: "ip2"
  bottom: "label"
  top: "loss"
}

the"cifar 10_quick_solver.prototx"is:

net: "C:/Users/Administrator/Desktop/data/train-test/cifar10_quick_train_test.prototxt"
test_iter: 2
test_interval: 50
base_lr: 0.001
momentum: 0.9
weight_decay: 0.004
lr_policy: "fixed"
display: 10
max_iter: 500
snapshot: 50
snapshot_format: HDF5
snapshot_prefix: "C:\Users\Administrator\Desktop\data"
solver_mode: CPU

Besides,train file has 50 pictures,25 cats and 25 birds
test file has 10 pictures,5 cats and 5 birds
 I've deal with them as  size of 28*28.

**Comment by Noiredd** [comment]
Check your second path - you have `/` instead of `\`, hence protobuf attempts to parse the `\U` in `C:\Users` as an escaped character, and fails.

Please do not post usage, installation, or modeling questions, or other requests for help to Issues. Use the [caffe-users list][1] instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe. Please read the [guidelines for contributing][2] before submitting an issue or a pull request.

 [1]: https://groups.google.com/forum/#!forum/caffe-users
 [2]: https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md
